aktuality/draft/index.html

```diff
@@ -335,6 +335,43 @@ <h4>Obrázek</h4>
 
                 </section>
 
+                <section id="vlastnost-aktualita-informuje-o">
+                    <h4>Informuje o</h4>
+                    <dl>
+                        <dt>
+                            Vlastnost
+                        </dt>
+                        <dd>
+                            <code>informuje_o</code>
+                        </dd>
+                        <dt>
+                            Typ
+                        </dt>
+                        <dd>
+                            <a href="../../základní-datové-typy/draft/#url">IRI</a>
+                        </dd>
+                        <dt>
+                            Jméno
+                        </dt>
+                        <dd>
+                            Informuje o
+                        </dd>
+                        <dt>
+                            Popis
+                        </dt>
+                        <dd>
+                            IRI datové entity, vytvořené dle jiné Otevřené formální normy(OFN), na kterou se aktualita váže. Typicky se jedná například o akci o které aktualita informuje, která byla ale zároveň vytvořená dle OFN Akce.
+                        </dd>
+                        <dt>
+                            Příklady
+                        </dt>
+                        <dd>
+                            <code>https://www.unesco-czech.cz/900_594766_trebic-mym-objektivem/#page_start</code>
```

Comment:
@spalrobert Ale tohle je přesně URL webové stránky, které to být nemá. Prosím o opravu na IRI nějaké entity, kterou máme třeba v příkladech jiných OFN